.github/pull_request_template.md

```diff
@@ -0,0 +1,9 @@
+## Describe your changes
+
+## Issue ticket number and link
+
+## Checklist before requesting a review
+- [ ] I have performed a self-review of my code
+- [ ] My code follows the style guidelines of this project
+- [ ] I have made corresponding changes to the documentation
```

Comment:
We could add a `CONTRIBUTING.md` explaining things like DCO and let the PR creator click a checkbox here that they have a read that:

```suggestion
- [ ] I have made corresponding changes to the documentation
- [ ] I read the [contribution documentation](https://github.com/EVerest/libocpp/blob/main/CONTRIBUTING.md) and
  made sure that:
  - My commits are signed-off to accept the DCO.
```